firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -212,7 +212,7 @@ class BrowserViewController: UIViewController,
 
     private lazy var effect: some UIVisualEffect = {
 #if canImport(FoundationModels)
-        if #available(iOS 26, *), !DefaultBrowserUtil.isRunningLiquidGlassEarlyBeta {
+        if #available(iOS 26, *), !DefaultBrowserUtility.isRunningLiquidGlassEarlyBeta {
```

Comment:
nitpick not related to your PR, but I don't think `isRunningLiquidGlassEarlyBeta` should live under `DefaultBrowserUtility`. `GlassUI` isn't related to the default browser stuff. Maybe into `AppInfo` would make more sense. We can create another ticket for this maybe (I don't mind doing that)! What do you think?